pkg/config/apiserver.go

```diff
@@ -251,7 +251,7 @@ func (a *API) Run() {
 
 // getScopes returns the scopes
 // "/scopes"
-func (a *API) getScopes(req *restful.Request, resp *restful.Response) {
+func (a *API) getScopes(_ *restful.Request, resp *restful.Response) {
```

Comment:
If any parameter is named, all parameters must be named - you can't mix and match. Since we use `resp` in the body, the first parameter must have a name. In the other cases they were fakes used in tests that ignored their parameters, so we could remove the names of all of them.